examples/ol/main.js

```diff
@@ -0,0 +1,28 @@
+import Map from 'ol/Map.js';
+import View from 'ol/View.js';
+import MVT from 'ol/format/MVT.js';
+import VectorTileLayer from 'ol/layer/VectorTile.js';
+import OGCVectorTile from 'ol/source/OGCVectorTile.js';
+
+const map = new Map({
+  target: 'map',
+  layers: [
+    new VectorTileLayer({
+      source: new OGCVectorTile({
+        url: 'http://127.0.0.1:8081/collections/public.countries/tiles/WebMercatorQuad',
+        context: {"z": "tileMatrix", "y": "tileRow", "x": "tileCol"},
```

Comment:
@francbartoli do you know how it works to tell ol to use `Z/X/Y` instead of `tileMatrix/tileRow/tileCol` ?